### ISSUE TITLE
Software with number in name

### ISSUE THREAD

**Issue by jameshowison** [howisonlab/softcite-dataset#624]
I'm noticing a few like SARF2 or ggplot2 where there is a number in the name.  My thinking here is that the most consistent way to treat those is that the numbers are part of the software name and not a version number.

What do you @caifand and @kermitt2 think?  Hey, there's a number in kermitt2 :)

**Comment by caifand** [comment]
Yes. I did not clean those. This is more common in bioinformatic tools (which is an important category of research software to us, I believe?). It's like blast/blastn, or clustalw/clustalx, or bowtie/bowtie 2. There are different flavors of software derived from the same lineage. Not a piece of software that has been updated in a linear manner over time.
